src/txdb.cpp

```diff
@@ -12,7 +12,7 @@
 #include <uint256.h>
 #include <util.h>
 #include <ui_interface.h>
-#include <init.h>
+#include <shutdown.h>
```

Comment:
Keep sorted.

Reply:
Note that you can install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script which fixes all white space and sorting issues